sql/tidb-specific.md

```diff
@@ -294,6 +318,37 @@ TiDB 在 MySQL 的基础上，定义了一些专用的系统变量和语法用
 这个变量用来设置是否禁用显式事务自动重试，设置为 1 时，不会自动重试，如果遇到冲突需要在应用层重试。
 是否需要禁用自动重试，请参考[自动重试的风险](./transaction-isolation.md#乐观事务注意事项)
 
+## tidb_enable_table_partition
+
+作用域：SESSION
+
+默认值：0
+
+这个变量用来设置是否开启 TABLE PARTITION 特性。
+
+## tidb_backoff_lock_fast
+
+作用域：SESSION | GLOBAL
+
+默认值：100
+
+这个变量用来设置读请求遇到锁的 backoff 时间
```

Comment:
Please also add a period in the end of the sentence, to keep it consistent with others.